README.md

```diff
@@ -21,7 +21,7 @@ This is a port of [babyagi](https://github.com/yoheinakajima/babyagi) with [Lang
 - [x] Execution history
 - [x] Faster speeds and fewer errors. ([😺 BabyCatAGI](https://twitter.com/yoheinakajima/status/1657448504112091136))
 - [ ] Display the current task and task list
-- [ ] i18n support
+- [x] i18n support
```

Comment:
🙌 🙌

Reply:
There are no missing keys at the moment and the duplicated loading I think I've fixed. No Hydration errors.